src/views/Project/index.jsx

```diff
@@ -0,0 +1,37 @@
+import { Component } from 'react';
+import Typography from 'material-ui/Typography';
+import ReactMarkdown from 'react-markdown';
+import projects from '../../data/loader';
+import BugsTable from '../../components/BugsTable';
+
+class Project extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
You don't need to have a constructor here. You can initialize the state as a class property:

```jsx
export default class Project extends Component {
  state = {
    fileName: this.props.match.params.projectKey,
    projects,
  };

// ...
```